.github/workflows/ci.yml

```diff
@@ -80,8 +70,23 @@ jobs:
             ./crates/node/index.d.ts
           key: ${{ runner.os }}-oxide-${{ hashFiles('./crates/**/*') }}
 
-      # Nx Cloud start-ci-run
-      - run: npx nx-cloud start-ci-run --distribute-on="3 my-linux-rust-large" --stop-agents-on-failure=false --stop-agents-after="integration-test"
+      # Install Rust
+      - name: Install Rust (Stable)
+        run: rustup default stable
+
+      - name: Setup rust target
+        run: rustup target add ${{ matrix.target }}
```

Comment:
Is this needed? If we are running directly on mac/windows/linux, should this be picked up automatically?